### PR TITLE
Moving the Contact Us to a separate page with form

### PR DESCRIPTION
**Pull request by zacharytamas**
- [x] Create form with design
- [x] Hook up form validation
- [x] Make form actually submit

### CODE REVIEW

File: blog/index.html [hunk 1]
```diff
@@ -27,3 +27,5 @@ <h1 class="centered">The Latest From Our Blog</h1>
   </div>
   </div>
 </div>
+
+<link rel="import" href="/elements/elements.html">
```
Comment (zacharytamas) [inline blog/index.html]: The blog index still needs the tile component.

File: hire-us.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+<div class="section-container">
+  <div class="section-content">
+    <h2>Hire Us</h2>
+    <contact-us-form id="hireUsForm"></contact-us-form>
+    <p id="hireUsThankYou">
+    </p>
+  </div>
+</div>
+<nl-contact-us></nl-contact-us>
```
Comment (zacharytamas) [inline hire-us.md]: Why are we preserving the modal? I had changed the places that would have launched it before to go to the new page.

Comment (adamduren) [inline hire-us.md]: Whoops that was just for testing. Good find. Should I remove the modal file as well?

**Comment by zacharytamas** [comment]
👍

**Pull request merged**